interpreters/lua/Kconfig

```diff
@@ -0,0 +1,90 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig INTERPRETERS_LUA
+	bool "Lua interpreter support"
+	default n
+	select ARCH_SETJMP_H
+	select LIBC_FLOATINGPOINT
```

Comment:
can we let user select LIBC_FLOATINGPOINT?

Reply:
If all Lua number represent as float, the selection look reasonable. If Lua support integer number is better to let user make the final decision.